README.md

```diff
@@ -12,13 +12,16 @@ This information can be useful, for example, to companies deciding whether to in
 
 The [Revised Agreement on Government Procurement](https://www.wto.org/english/docs_e/legal_e/rev-gpr-94_01_e.htm) (GPA) includes: "each notice of intended procurement shall include … c. for recurring contracts, an estimate, if possible, of the timing of subsequent notices of intended procurement".
 
-The European Union is a [party](https://www.wto.org/english/tratop_e/gproc_e/memobs_e.htm) to the GPA, and as such its [Directive 2014/24/EU](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2014.094.01.0065.01.ENG) (Public contracts — setting out clear ground rules) includes: "Part C: Information to be included in contract notices … 27. In the case of recurrent procurement, estimated timing for further notices to be published."
+The European Union is a [party](https://www.wto.org/english/tratop_e/gproc_e/memobs_e.htm) to the GPA, and as such its [Directive 2014/24/EU](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2014.094.01.0065.01.ENG) (Public contracts — setting out clear ground rules) includes: "Part C: Information to be included in contract notices … 27. In the case of recurrent procurement, estimated timing for further notices to be published".
```

Comment:
Keep the period inside the quotation marks (American English style).

Reply:
These Americans... Done!